EventListener/EmailSubscriber.php

```diff
@@ -65,7 +69,7 @@ public function onEmailGenerate(Events\EmailSendEvent $event)
         $event->setSubject($subject);
 
         $content = $this->templateProcessor->processTemplate($content,  $event->getLead());
-        $event->setContent($content);
+        $event->setContent($content, false);
```

Comment:
I don't see where this optional param is coming from in the mautic source. 

https://github.com/mautic/mautic/blob/4.4/app/bundles/EmailBundle/Event/EmailSendEvent.php#L156